src/routes/contents/[id]/+page.svelte

```diff
@@ -0,0 +1,229 @@
+                contentsStack.pushItem($page.data.menu.id);
+                setReference(item);
+                goto(`${base}/`);
+                break;
+            //need to check if these are the correct strings to use
+            case 'screen':
+                //not handled yet
+                //goes to another contents page
+                contentsStack.pushItem($page.data.menu.id);
+                await goto(`${base}/contents/${item.linkTarget}`);
+                break;
+            case 'other':
+                //not handled yet
+                //switch on item.linkLocation
+                switch (item.linkLocation) {
+                    case 'about':
+                    case 'settings':
+                        goto(`${base}/${item.linkLocation}`);
+                        break;
+                    case 'layout':
+                        modal.open(MODAL_COLLECTION);
+                        //also the title for this one says &amp; instead of & in the json
+                        break;
+                    case 'website':
+                        //opens in a separate tab
+                        window.open(`${item.linkTarget}`, '_blank');
+                        break;
+                }
+                break;
+
+            default:
+                // For other book types (e.g. quiz), the linkType will be
+                // the book type and the linkLocation will have the route
+                // to the viewer of the book type.
+                goto(`${base}/${item.linkLocation}`);
+                break;
+        }
+    }
+
+    function setReference(item) {
+        // reference = formatReferenceTarget(item.linkTarget)
+        let book;
+        let chapter;
+        let verse;
+        const reference = item.linkTarget.split('.');
+
+        switch (reference.length) {
+            case 1:
+                book = reference[0];
+                break;
+            case 2:
+                book = reference[0];
+                chapter = reference[1];
+                break;
+            case 3:
+                book = reference[0];
+                chapter = reference[1];
+                verse = reference[2];
+                break;
+            default:
+                break;
+        }
+        refs.set({
+            book: book,
+            chapter: chapter,
+            verse: verse
+        });
+    }
+
+    //set the title for the current contents page
+    function setTitle(page) {
+        //checks title type and returns the appropriate title or lack of title
+
+        /* title options:
+            - app-name 
+            - screen (current contents screen title)
+            - custom 
+            - none
+        
+            I also noticed that if you build files with a custom 
+            title and then again without specifying a title aka 
+            any of the other options, the title in the json was the 
+            previously used custom title, is this ok?
```

Comment:
It is ok.  You might change the comment to note that the custom title might be in the JSON even though it isn't currently a custom title type.  ... or we could change the convertContents to not pass it along (marginal value in doing this though)